Tests/Opc.Ua.Core.Tests/Opc.Ua.Core.Tests.csproj

```diff
@@ -19,10 +19,12 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
   </ItemGroup>
-
+  
   <ItemGroup>
+    <PackageReference Include="Portable.BouncyCastle" Version="1.8.8" />
```

Comment:
Only for TargetFramework is net462 required

Reply:
good catch!